src/api/categories/adaptors/index.ts

```diff
@@ -228,29 +217,33 @@ export const getChainPageData = async (type: string, chain?: string): Promise<IO
 	const revenueProtocols =
 		type === 'fees'
 			? feesOrRevenue?.protocols?.reduce(
-					(acc, protocol) => ({ ...acc, [protocol.name]: protocol }),
-					{} as IJSON<ProtocolAdaptorSummary>
-			  ) ?? {}
+				(acc, protocol) => ({ ...acc, [protocol.name]: protocol }),
+				{} as IJSON<ProtocolAdaptorSummary>
+			) ?? {}
 			: {}
 
 	const { parentProtocols } = protocolsData
 	const parentProtocolsMap = parentProtocols.reduce((acc, curr) => ({ ...acc, [curr.id]: curr }), {})
 
 	const protocolsWithSubrows = protocols.reduce((acc, protocol) => {
+		if (!protocol?.name) return acc
+
 		// Assign mainrow and sub-row if has any
-		const slugName = sluggify(protocol?.name)
+		const slugName = sluggify(protocol.name)
 
 		let mainRow: undefined | IOverviewProps['protocols'][number] = undefined
 		let subRow: undefined | IOverviewProps['protocols'][number]['subRows'][number] = null
-		if (parentProtocolsMap[protocol.parentProtocol]) {
+
+		if (protocol.parentProtocol && parentProtocolsMap[protocol.parentProtocol]) {
 			mainRow = parentProtocolsMap[protocol.parentProtocol]
+			if (!protocol.total24h) protocol.total24h = 0
```

Comment:
?

Reply:
there was a bug if protocol was just listed or if it had no records, total24h was not returned/returned as undefined, and for some reason, the ended up on the top of the table because we sort by default on this field. This line is to ensure it remains at the bottom